docs/tvl.md

```diff
@@ -13,19 +13,32 @@ As you can see there are three types of value:
 Everything you see here is achieved via proper configuration files in our repository.
 
 ## Token guide
-
-1. Go to `config\src\tokens\source.jsonc`
+1. Go to `config\src\tokens\tokens.jsonc`
 2. Add your token to the proper chain
 
-   - required fields are: **symbol & address**
-   - optional fileds are: type, formula, category, bridgedUsing & coingeckoId
-   - for `ethereum` the `type` and `formula` are `CBV` and `locked` by default, for other chains you need to specify it
-   - `category` is `other` by default
-   - if you do not see your chain run `yarn coingecko:platforms` (TODO: update docs when dynamic chains functionality is merged)
+→ Canonically bridged tokens (CBV) should always be added to Ethereum.
+- The required fields are `symbol` **&** `address`.
+- Optional fields are: `type`, `formula`, `category`, `bridgedUsing` & `coingeckoId`.
+- By default, the `type` is `CBV`, the `formula` is `locked`, and the `category` is `other`.
 
+→ Externally bridged tokens (EBV) and natively minted tokens (NMV) should be added to the token list of a specific L2.
+- If the chain is missing in the file, you have to add it
+- The required fields are symbol, address, type, formula. The optional fields are `category`, `bridgedUsing` & `coingeckoId`.
+    - If the token is externally bridged, you have to add a `bridgedUsing` object, containing at least the `bridge` = the bridge name. If we have it on our website, you can also add a `slug`.
+    - Make sure to run `yarn coingecko:platforms`. If the chain is not available there, you need to also specify the `coingeckoId`.
 3. Save the file
-4. Set environmental variable `<CHAIN>_RPC_URL`
-5. Run `yarn tokens`
-6. Commit changes & open the PR
 
-**Never edit generated.json until you know what you are doing**
+4. If you’re adding externally bridged or natively minted tokens, go to the .ts file of the chain those are added to in `config\src\layer2s`
+    - Check if there is `chainConfig` defined within `export const projectName: Layer2 = {}`
+    - If not, it should be added with the following information:
+        - `name` of the chain - should be the same as defined in the `tokens.jsonc` = the name of the .ts file
+        - `chainId`
+        - `explorerUrl`
+        - `explorerApi{}`, containing `url` and `type`
+        - `multicallContracts[]` - could be empty if not deployed
+        - `minTimestampForTvl` - the timestamp of the first/second block on the chain
```

Comment:
The information about adding `coingeckoPlatform` and running `yarn coingecko:platforms` should go here.